.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Discord
+    url: https://discord.gg/YJtPqVntqM
+    about: Please ask and answer questions here. This is also the place to report security vulnerabilities.
```

Comment:
Can we change `name` to "Questions and help" or something? Questions, concerns, community? We should aim to draw the eyes of people with questions, and `Discord` requires then to think extra (and that's bad, as we're trying to guide low-effort users with this in addition to uncaught cases).